src/frontend/src/handler/create_table.rs

```diff
@@ -850,7 +854,7 @@ fn derive_connect_properties(
     let mut connect_properties = source.properties.clone();
     if let Some(connector) = source.properties.get(UPSTREAM_SOURCE_KEY) {
         let table_name = match connector.as_str() {
-            MYSQL_CDC_CONNECTOR => {
+            "mysql-cdc" => {
```

Comment:
I would still suggest making them constants. Pattern matching on constants still works but we need to first **`use` them**.